### PR TITLE
chore: Update rubocop Jan 2022

### PR DESCRIPTION
**Pull request by zverok**
New cops checked on hubstaff-server, its PR will follow

## Related issues

- Source: —
- UAT: —
- QA: —
- Review app:—

### Development

- [x] The commit message follows our [guidelines](https://docs.hubstaff.com/hubstaff-docs/latest/great_commit_messages.html)
- [x] I have performed a self-review of my own code
- [x] I have thoroughly tested the changes
- [x] I have added tests that prove my fix is effective or that my feature works

### Security

- [x] Security impact of change has been considered

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -52,6 +57,15 @@ Lint/AmbiguousBlockAssociation:
   Exclude:
     - spec/**/*
 
+# Requires to change `foo + bar * baz` into `foo + (bar * baz)` for clarity. WHAT.
```
Comment (ayarotsky) [inline config/default.yml:60]: 😄

**Review by ayarotsky — APPROVED** [review]
👍🏼

File: config/default.yml [hunk 2]
```diff
@@ -22,6 +22,11 @@ AllCops:
...
     - 'node_modules/**/*'
 
+# Checks that the gemspec has metadata to require MFA from RubyGems
+# Irrelevant for internal gems
+Gemspec/RequireMFA:
+  Enabled: false
+
 # region Layout
 Layout/AssignmentIndentation:
   IndentationWidth: 4
```
Comment (ka8725) [inline config/default.yml:32]: Since we are here, can we update this as well?

https://netsoftllc.slack.com/archives/C02AMSXHH2P/p1638869672054200

Comment (zverok) [reply config/default.yml:32]: I'd rather not. Related server PR is big already, and it is after I extracted the follow-up about `FactoryBot.`
Changing this setting should lead to a separate PR (which, I imagine, will affect thousands of files)

**Pull request merged**